Scheduler: Add Appointment Edit Form Customization Topics (#8188)
DevExpress/devextreme-documentation: concepts/05 UI Components/Scheduler/030 Appointments/070 Appointment Edit Form/10 Customize the Appointment Edit Form.md

    import { DxScheduler, type DxSchedulerTypes } from 'devextreme-vue/scheduler';

    function handleAppointmentFormOpening(e: DxSchedulerTypes.AppointmentFormOpeningEvent) {
        e.form.getEditor('subjectEditor').option('disabled', true);
        e.form.itemOption('mainGroup.descriptionGroup.descriptionEditor', 'isRequired', true);
    }

    </script>

##### React

    <!-- tab: App.tsx -->
    import { Scheduler, type SchedulerTypes } from 'devextreme-react/scheduler';

    function handleAppointmentFormOpening(e: SchedulerTypes.AppointmentFormOpeningEvent) {
        e.form.getEditor('subjectEditor').option('disabled', true);
        e.form.itemOption('mainGroup.descriptionGroup.descriptionEditor', 'isRequired', true);
    }

    function App() {
        return (
            <Scheduler ...
                onAppointmentFormOpening={handleAppointmentFormOpening}
            />
        );
    };

---

[note] Specify full item paths described in [Predefined Items](/Documentation/Guide/UI_Components/Scheduler/Appointments/Appointment_Edit_Form/#Predefined_Items) as the **id** parameter in [itemOption()](/Documentation/ApiReference/UI_Components/dxForm/Methods/#itemOptionid).

> To customize the appointment edit form further, specify the **editing**.[form](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/form/) and **editing**.[popup](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/#popup) configuration objects. These objects configure the appointment edit form's Popup and Form components. The following code snippet implements an increased width for the Popup and the "static" label mode for the Form:

Reviewer: ```suggestion
To customize the appointment edit form further, specify the **editing**.[form](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/form/) and **editing**.[popup](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/#popup) configuration objects. These objects configure the appointment edit form's Popup and Form components. The following code snippet increases the width of the Popup and sets the "static" label mode for the Form:
```
